test/docker-compose.yml

```diff
@@ -0,0 +1,83 @@
+version: '2.0'
```

Comment:
Any reason you chose `2.0` and not the latest (`3.8`)?

Reply:
Good question, I copy pasted this file from the [builder's docker-compose.yml](https://github.com/rstudio/r-builds/blob/master/builder/docker-compose.yml) and didn't think about it. Looking up the version field now, it appears to have been deprecated and made optional in docker-compose.yml, so I'll try removing it.

https://docs.docker.com/compose/compose-file/#compose-file
> The Compose file is a [YAML](http://yaml.org/) file defining [version](https://docs.docker.com/compose/compose-file/#version-top-level-element) (DEPRECATED),

https://docs.docker.com/compose/
> A docker-compose.yml looks like this:
> ```yaml
> version: "3.9"  # optional since v1.27.0
> ```